### DISCUSSION TITLE
Best way to expose Swift library

### DISCUSSION THREAD

**Discussion by Techbossru** [NativeScript/NativeScript#9689] [Q&A]
### Issue Description

Im readed docs (https://v7.docs.nativescript.org/plugins/use-native-ios-libraries#apis-written-in-swift)

But can i really use Swift CocoaPod in Nativescript plugin?

CocoaPod im trying to use: Mindbox (https://github.com/mindbox-moscow/ios-sdk)

Im really stucked

### 

- [X] I have searched the [existing issues](https://github.com/NativeScript/NativeScript/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [ ] This is a bug report
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by triniwiz** [comment]
If the pod extends using `NSObject` it much easier to use in objc which this pod does , you can try `ns typings ios` it will generate the typings in `./typings` there should be a file containing the name `Mindbox` in one of the directories

**Reply by Techbossru** [reply]
pod not extending NSObject, can i do it manually?
`ns typings ios`
im doing it, but there are no methods that i want, in directory

**Reply by triniwiz** [reply]
I would create a lite swift wrapper exposing what I need to objc
